.gitignore

```diff
@@ -12,3 +12,4 @@ test-archives
 /test-results/
 /playwright-report/
 /playwright/.cache/
+.vscode/**
```

Comment:
We have such a folder (non-git-ignored) for `chromatic` and `chromatic-cli` repos, so I dunno that we want to ignore it here. 

@jwir3 did files autogenerate into here?

Reply:
Ah, yeah, that's a good question. It showed on my machine as a new folder for some reason. I'll remove the line from `.gitignore`.